tmt/steps/execute/__init__.py

```diff
@@ -619,11 +636,19 @@ def load_custom_results(self, invocation: TestInvocation) -> list["tmt.Result"]:
                 result=ResultOutcome.ERROR)]
 
         custom_results = []
+        # Check whether a result exists for the parent test.
```

Comment:
It shouldn't exist: according to documentation, a result representing the parent test itself is supposed to be named `/`, every other name would be prefixed with the test name. Why new handling for results named after the test name?

Reply:
I see this bit in `tmt-report-results`:

```shell
# If TESTNAME is just / then set it to the parent test name.
[[ $TESTNAME = "/" ]] && TESTNAME=$TMT_TEST_NAME
```

But that applies to results reported by `tmt-report-result` and loaded via `load_tmt_report_results()`, while `load_custom_results()` should never encounter a result named after the parent test, given the documentation on custom test results.

I suppose the test author may use `tmt-report-result` to construct custom results, but then we violate the documented assumption that there's a result named `/` which represents the test itself. In a custom result file, constructed by our tool, would be no such result, because its name `/` would be rewritten. This seems like a step away from the documentation, and seems like an inconsistency to me - custom results created manually by users would follow this rule, but those created by our tool would not.

Can you provide more info why this particular name rewrite is needed?

Reply:
I had thought one of the previous comments was requesting that change, but it appears that I'd misread it.

I've removed that bit of logic.

Here's the log with the most recent results of test ouput: 
[testing.txt](https://github.com/user-attachments/files/15544897/testing.txt)